tests/test_conversation_utils.py

```diff
@@ -52,14 +50,14 @@ def test_truncate_message_last_large(self):
         big_chat_message.content = big_chat_message.content + "\n" + "Question?"
         copy_big_chat_message = big_chat_message.copy()
 
-        chat_messages.append(big_chat_message)
+        chat_messages.insert(0, big_chat_message)
         tokens = sum([len(self.encoder.encode(message.content)) for message in chat_messages])
 
         prompt = utils.truncate_messages(chat_messages, self.max_prompt_size, self.model_name)
         tokens = sum([len(self.encoder.encode(message.content)) for message in prompt])
 
         # The original object has been modified. Verify certain properties
-        assert len(chat_messages) < 26
-        assert len(chat_messages) > 1
+        assert len(prompt) < 26
+        assert len(prompt) > 1
```

Comment:
Based on the comment above, we should continue to check if `chat_messages` was updated in-place as well, right?